content/tutorials/build-apps/build-a-desktop-wallet-in-js.md

```diff
@@ -0,0 +1,211 @@
+
+Now that we have created the home page and the send XRP page, let's create the transactions page that will display the transaction history of the account.
+
+  1. Account: The account that sent the transaction.
+  2. Destination: The account that received the transaction.
+  3. Amount: The amount of XRP sent in the transaction.
+  4. Transaction Type: The type of transaction.
+  5. Result: The result of the transaction.
+  6. Link: A link to the transaction on the XRP Ledger Explorer.
+
+![Transactions Page Screenshot](img/js-wallet-transaction.png)
+
+1. Create a folder named transaction-history in the src directory.
+
+2. Add two files named transaction-history.js and transaction-history.html.
+
+3. In the transaction-history.js file, copy the code written below.
+
+{{ include_code("_code-samples/build-a-wallet/js/src/transaction-history/transaction-history.js", language="js") }}
+
+In the code above we fetch the transactions from the XRP Ledger by using the [account_tx](https://xrpl.org/account_tx.html) method. You can read more about the account_tx method [here](https://xrpl.org/account_tx.html).
+
+4. Copy the contents of the [transaction-history.html](_code-samples/build-a-wallet/js/src/transaction-history/transaction-history.html) file to your transaction-history.html file.
+
+You can use this code as a starting point and customize it to suit your needs. For example, you might want to add more columns to the table. You can also check out the [XRP Ledger Explorer](https://livenet.xrpl.org/) to see how the transaction details are displayed.
+
+### Step 5: Running the App
+
+A wallet app is not complete without a way to run it. In this step, we will set up the vite bundler to run the app.
+
+1. Set up vite bundler, copy the code written below to the vite.config.js file.
+
+{{ include_code("_code-samples/build-a-wallet/js/vite.config.js", language="js") }}
+
+We've used Vite to bundle the app, if you're unfamiliar with Vite, you can check out the [Vite documentation](https://vitejs.dev/guide/) to learn more.
+
+xrpl.js requires some configuration to run with the vite bundler. You can find the configuration in the code below or you can check out the [xrpl.js documentation](https://github.com/XRPLF/xrpl.js/blob/main/UNIQUE_SETUPS.md#using-xrpljs-with-vite-react).
+
+
+2. Add script to package.json
+
+In your package.json file, add:
+
+```json
+"scripts": {
+    "dev": "vite"
+}
+```
+
+Now our app is ready to run. To run the app, open terminal in the project directory and run:
+
+```bash
+yarn dev ## or npm run dev
+```
+
+This will start the app in development mode. Check your terminal for the URL where the app is running. Click the URL to open the app in your browser. You should see the home page of the app. 
+
+If you ran into trouble while running the app, go over the steps again to ensure you followed them correctly. If still the app doesn't run, feel free to ask on XRPL Community [Discord](https://discord.com/invite/KTNmhJDXqa).
```

Comment:
I'd recommend removing the suggestion to re-read the steps because that can seem a bit condescending (it's usually what people try first)

```suggestion
If you get stuck building while doing this tutorial, or working on another project, feel free to ask for help in the XRPL's [Developer Discord](https://discord.com/invite/KTNmhJDXqa).
```